static/src/stylesheets/module/_release-message.scss

```diff
@@ -432,7 +432,8 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
 .site-message--membership-message,
 .site-message--membership-message-uk,
 .site-message--membership-message-uk-redisplayed,
-.site-message--membership-message-usa {
+.site-message--membership-message-us,
+.site-message--membership-message-us-redisplayed {
```

Comment:
You might also want to remove switch for this AB test. AB tests are always behind the switch.